src/js/hca-rjsf/helpers.jsx

```diff
@@ -0,0 +1,10 @@
+import { transformForSubmit } from '../common/schemaform/helpers';
+
+export function transform(formConfig, form) {
+  const formData = transformForSubmit(formConfig, form);
+  return JSON.stringify({
+    educationBenefitsClaim: {
```

Comment:
I think it's just called `form` for hca: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/js/hca/components/HealthCareApp.jsx#L147